packages/common/react-components/src/components/Table/Table.tsx

```diff
@@ -5,7 +5,7 @@
 import React, { useMemo } from 'react';
 import { Column, useFlexLayout, useResizeColumns, useTable } from 'react-table';
 
-import { mx } from '@dxos/react-components';
+import { mx } from '../../util';
 
 // https://github.com/TanStack/table/blob/v7/examples/full-width-resizable-table/src/App.js
 
```

Comment:
This component is missing a ton of keyboard access and accessibility features, but we can address that later. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/grid_role